src/caremote.c

```diff
@@ -47,6 +47,8 @@ struct CaRemote {
         int input_fd;
         int output_fd;
 
+        int rate_limit;
```

Comment:
hmm, so the type should probably be uint64_t here... "int" definitely sounds wrong to me, as it is signed... size_t also sounds wrong to me, as this isn't bytes in memory, but just the general concept of bytes.... (or in other words: the size of this shouldn't be different if you compile this on x86 or x86-64...)

Also, can you add a suffix "_bps" to this, to clarify this is in bytes?